src/action.js

```diff
@@ -105,6 +107,40 @@ async function commit(params) {
   return [treeResponse.data.sha, commitResponse.data.sha];
 }
 
+async function getQuestionData(titleSlug, leetcodeSession) {
+  log(`Getting question data for ${titleSlug}...`);
+
+  const headers = {
+    "Content-Type": "application/json",
+    "Cookie": `LEETCODE_SESSION=${leetcodeSession};`
+  }
+
+  const graphql = JSON.stringify({
+    query: `query getQuestionDetail($titleSlug: String!) {
+      question(titleSlug: $titleSlug) {
+        content
+      }
+    }`,
+    variables: {"titleSlug": titleSlug},
+  })
+
+  const requestOptions = {
+    method: 'POST',
+    headers,
+    body: graphql,
+    redirect: 'follow'
+  };
+
+  try {
+    const response = await fetch("https://leetcode.com/graphql/", requestOptions);
```

Comment:
Is it possible to re-use axios here for consistency? I'm not actually too familiar with node-fetch, but it seems like it might be worth migrating to using that in the future.